packages/transaction-controller/src/TransactionController.test.ts

```diff
@@ -4428,12 +4451,18 @@ describe('TransactionController', () => {
           lastFetchedBlockNumbers: {},
         };
 
+        const controller = newController({
+          // TODO: Replace `any` with type
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          state: mockedControllerState as any,
+        });
+
```

Comment:
Just curious why did this need to need to move up here?

Reply:
because it (re)sets the global mockMessenger variable that we spy on below. Previously this occured in the beforeEach so it didn't matter when the mockMessenger.call override occurred in this scenario